modules/cert-manager-configure-cloud-credentials-aws-sts.adoc

```diff
@@ -0,0 +1,167 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-authenticate-aws.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-configure-cloud-credentials-aws-sts"]
+= Configuring cloud credentials for the {cert-manager-operator} for the AWS Security Token Service cluster
+
+To configure the cloud credentials for the {cert-manager-operator} on the AWS Security Token Service (STS) cluster you must use the `ccoctl` binary to generate the cloud credentials manually and apply it on the cluster.
+
+.Prerequisites
+
+* You must extract and prepare the `ccoctl` binary.
+* The {cert-manager-operator} 1.11.1 or later is installed.
+
+.Procedure
+
+. Create a directory to store the `CredentialsRequest` custom resource (CR) by running the following command:
++
+[source,terminal]
+----
+$ mkdir credentials-request
+----
+
+. Change the directory by running the following command:
++
+[source,terminal]
+----
+$ cd credentials-request
+----
+
+. Create a `CredentialsRequest` resource YAML file, for example, `sample-credential-request.yaml`, as follows:
++
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: cert-manager
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+    - action:
+      - "route53:GetChange"
+      effect: Allow
+      resource: "arn:aws:route53:::change/*"
+    - action:
+      - "route53:ChangeResourceRecordSets"
+      - "route53:ListResourceRecordSets"
+      effect: Allow
+      resource: "arn:aws:route53:::hostedzone/*"
+    - action:
+      - "route53:ListHostedZonesByName"
+      effect: Allow
+      resource: "*"
+  secretRef:
+    name: aws-creds
+    namespace: cert-manager
+  serviceAccountNames:
+  - cert-manager
+----
+
+. Create a `CredentialsRequest` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sample-credential-request.yaml
+----
+
```

Comment:
Similar as GCP STS https://github.com/openshift/openshift-docs/pull/59895 , above manual CredentialsRequest creation for STS cluster is NOT needed.

Reply:
One more thing, then `cd credentials-request` is not needed.

Reply:
I suppose creating the YAML file under a `credentials-request` directory is not mandatory. To be consistent with other STS/workload identity PRs[1][2][3], I removed the first two steps i.e `mkdir credentials-request` and `cd credentials-request`.
[1]https://github.com/openshift/openshift-docs/pull/59895
[2]https://github.com/openshift/openshift-docs/pull/60433
[3]https://github.com/openshift/openshift-docs/pull/59907

Reply:
If it is mandatory I will update change accordingly